nmpc_rabbit_node:
  ros__parameters:
    initX: [0.0, 0.0, 0.0]
    initU: [0.0, 0.0, 0.0, 0.0]
    lowX: [-3.0, -3.0, -3.14]
    highX: [3.0,  3.0,  3.14]
    lowU: [-10, -10, -10, -10]
    highU: [10, 10, 10, 10]
    mat_Q: [75, 75, 90]
    mat_R: [0.1, 0.1, 0.1, 0.1]